examples/tutorial/metric-learning.ipynb

```diff
@@ -0,0 +1,668 @@
+{
```

Comment:
<p>near neighbours -&gt; either nearest neighbours or neighbours</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/objax/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
changed all references in text and code to "nearest neighbours"